### PR TITLE
(totalcommander) refactoring and finished migration

### PR DESCRIPTION
**Pull request by majkinetor**
Based upon work @pascalberger did (#501), I refactored a code and finished it. It will push new version on merge. 

Remains to be removed from @dtgm repo but since there were no recent updates I guess its safe to assume it wont cause any trouble to merge it ASAP.

### CODE REVIEW

File: automatic/totalcommander/TotalCommander.nuspec [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <metadata>
+    <id>totalcommander</id>
+    <title>Total Commander</title>
+    <version>0.0</version>
+    <authors>Christian Ghisler</authors>
+    <owners>chocolatey</owners>
```
Comment (pascalberger) [inline automatic/totalcommander/TotalCommander.nuspec]: Original owners missing

**Comment by majkinetor** [comment]
> Btw. you could also have done this changes on my original branch instead of creating a new branch & PR

A mistake, wont happen again.

File: automatic/totalcommander/TotalCommander.nuspec [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    <tags>file manager shareware nagscreen admin</tags>
+    <copyright>© 1995 Christian Ghisler</copyright>
+    <licenseUrl>http://www.ghisler.com/order.htm</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/4d9fd7c774e1ae7edb688d04ac9b55b87477e318/icons/totalcommander.png</iconUrl>
+    <releaseNotes>http://ghisler.com/whatsnew.htm</releaseNotes>
+    <dependencies>
+      <dependency id="chocolatey-core.extension" version="1.0.7" /> 
+      <dependency id="7zip.portable" />
+      <dependency id="autohotkey.portable" />
```
Comment (AdmiringWorm) [inline automatic/totalcommander/TotalCommander.nuspec]: as per [1.1.11 Dependency versions](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#1111-dependency-versions) please specify a minimum version for both 7zip.portable and autohotkey.portable

File: automatic/totalcommander/TotalCommander.nuspec [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+[Plugins](http://www.ghisler.ch/wiki/index.php/Developer%27s_corner)
+[Plugin downloads](http://totalcmd.net/)
+[Plugin development](http://totalcmd.net/directory/developer.html)
+    </description>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/totalcommander</packageSourceUrl>
+    <projectUrl>http://www.ghisler.com</projectUrl>
+    <bugTrackerUrl>http://www.ghisler.ch/wiki/index.php/Known_bugs_and_workarounds</bugTrackerUrl>
+    <docsUrl>http://www.ghisler.ch/wiki/index.php</docsUrl>
+    <mailingListUrl>http://www.ghisler.ch/board/index.php</mailingListUrl>
+    <tags>file manager shareware nagscreen admin</tags>
```
Comment (AdmiringWorm) [inline automatic/totalcommander/TotalCommander.nuspec]: missing trial tag?

**Comment by majkinetor** [comment]
Done

**Comment by majkinetor** [comment]
I will merge this now because Gallery version is late almost a year and there is almost zero probability it will cause problems with @dtgm because there is pending merge.

**Pull request merged**